src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ CHANGELOG
 
 5.4
 ---
+ * Fix `XmlFileLoader` not respecting when env for services
```

Comment:
New lines should be appended, not prepended. Also I'm not sure to understand the sentence. Maybe it should be rephrased to be clearer

Reply:
We can completely remove this change. Changelog entries for bugfixes are generated automatically. The changelog files in the individual packages contain new features/deprecations etc. introduced in minor releases.

Reply:
Ah yes of course

Reply:
Can see the confusion, had some difficulty putting the correct words to the bug. Have removed the line in general based on xabbuh comment.